gui/styles/_wizard.styl

```diff
@@ -148,9 +148,13 @@
       position absolute
       top .8em
       right 1em
+  &.step-error .folder__selector
+    background-color: lighten(red, 30%)
 
   .btn
     color #9164F5
 
 .step-error
   background-color red
+  .btn
+    color red
```

Comment:
Nope. The background of the page is red, and on this page, the button is white with font in red.